src/main/java/net/minecraftforge/registries/DataPackRegistriesHooks.java

```diff
@@ -57,4 +58,13 @@ public static Set<ResourceKey<? extends Registry<?>>> getSyncedCustomRegistries(
     {
         return SYNCED_CUSTOM_REGISTRIES_VIEW;
     }
+
+    /**
+     *  FORGE: add non-vanilla registry namespace to loader directory, same format as tag directory (see net.minecraft.tags.TagManager#getTagDir(ResourceKey))
```

Comment:
This is a crude copy of the comment from the patch. We are in a forge class, this does not need a forge prefix. It should also not have two spaces. And there should be a {@link} used instead of just parentheses. Also a blank line between the description and the tags `@param`, etc.